r2i/tensorflow/engine.cc

```diff
@@ -217,7 +217,21 @@ std::shared_ptr<r2i::IPrediction> Engine::Predict (std::shared_ptr<r2i::IFrame>
   }
 
   std::shared_ptr<TF_Tensor> pout_tensor (out_tensor, TF_DeleteTensor);
-  prediction->SetTensor (pgraph, out_operation, pout_tensor);
+
```

Comment:
I don't think we should enforce the memory copy, I don't like that approach. Each backend may implement an optimized way to do this.

Reply:
I've reverted this change.